src/main/java/by/Starleken/entities/User.java

```diff
@@ -0,0 +1,103 @@
+package by.Starleken.entities;
+
+import jakarta.persistence.*;
+
+import java.util.List;
+import java.util.Objects;
+
+@Entity
+@Table(name = "`users`")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "username")
+    private String username;
+
+    @Column(name = "imageURL")
+    private String imageURL;
+
+    //TODO
+    @Column(name = "role")
+    private String role;
+
+    @OneToMany(mappedBy = "user", fetch = FetchType.EAGER)
+    private List<Comment> comments;
+
+    public User() {
+    }
+
+    public User(Long id, String username, String imageURL, String role, List<Comment> comments) {
+        this.id = id;
+        this.username = username;
+        this.imageURL = imageURL;
+        this.role = role;
+        this.comments = comments;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public String getUsername() {
+        return username;
+    }
+
+    public void setUsername(String username) {
+        this.username = username;
+    }
+
+    public String getImageURL() {
+        return imageURL;
+    }
+
+    public void setImageURL(String imageURL) {
+        this.imageURL = imageURL;
+    }
+
+    public String getRole() {
+        return role;
+    }
+
+    public void setRole(String role) {
+        this.role = role;
+    }
+
+    public List<Comment> getComments() {
+        return comments;
+    }
+
+    public void setComments(List<Comment> comments) {
+        this.comments = comments;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        User user = (User) o;
+        return Objects.equals(id, user.id) && Objects.equals(username, user.username) && Objects.equals(imageURL, user.imageURL) && Objects.equals(role, user.role) && Objects.equals(comments, user.comments);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(id, username, imageURL, role, comments);
```

Comment:
It's better to exclude the collections with different entities from equals and hashcode

Remove `comments` in this case. And from `toString()` method too